{"uid":"f841dbf164847c70","name":"test_scenario_3_orders","fullName":"tests.tests_steps.TestScenario3orders#test_scenario_3_orders","historyId":"00f7485f22028d496ac230dc80af8a94","time":{"start":1659439730786,"stop":1659439730786,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561b0c0b6cd3 <unknown>\n#1 0x561b0bebe968 <unknown>\n#2 0x561b0bee325c <unknown>\n#3 0x561b0bede8fa <unknown>\n#4 0x561b0bf1994a <unknown>\n#5 0x561b0bf13aa3 <unknown>\n#6 0x561b0bee93fa <unknown>\n#7 0x561b0beea555 <unknown>\n#8 0x561b0c0fe2bd <unknown>\n#9 0x561b0c102418 <unknown>\n#10 0x561b0c0e836e <unknown>\n#11 0x561b0c103078 <unknown>\n#12 0x561b0c0dcbb0 <unknown>\n#13 0x561b0c11fd58 <unknown>\n#14 0x561b0c11fed8 <unknown>\n#15 0x561b0c139cfd <unknown>\n#16 0x7ffa901b0609 <unknown>","statusTrace":"@pytest.fixture(scope=\"class\")\n    def start_session():\n>       Session.start()\n\nconftest.py:77: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests_utils/session.py:27: in start\n    cls._driver = cls.get_new_driver()\ntests_utils/session.py:43: in get_new_driver\n    driver = webdriver.Chrome(ChromeDriverManager().install(), options=options)\n/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py:69: in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py:92: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:277: in __init__\n    self.start_session(capabilities, browser_profile)\n/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:370: in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:435: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f66c06f3ee0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: Chrome failed to start: exited a...\\\\n#14 0x561b0c11fed8 \\\\u003Cunknown>\\\\n#15 0x561b0c139cfd \\\\u003Cunknown>\\\\n#16 0x7ffa901b0609 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        Checks that a JSON response from the WebDriver does not have an error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get('status', None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get('value', None)\n            if value_json and isinstance(value_json, str):\n                import json\n                try:\n                    value = json.loads(value_json)\n                    if len(value.keys()) == 1:\n                        value = value['value']\n                    status = value.get('error', None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get('message')\n                    else:\n                        message = value.get('message', None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        if status in ErrorCode.NO_SUCH_ELEMENT:\n            exception_class = NoSuchElementException\n        elif status in ErrorCode.NO_SUCH_FRAME:\n            exception_class = NoSuchFrameException\n        elif status in ErrorCode.NO_SUCH_SHADOW_ROOT:\n            exception_class = NoSuchShadowRootException\n        elif status in ErrorCode.NO_SUCH_WINDOW:\n            exception_class = NoSuchWindowException\n        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:\n            exception_class = StaleElementReferenceException\n        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:\n            exception_class = ElementNotVisibleException\n        elif status in ErrorCode.INVALID_ELEMENT_STATE:\n            exception_class = InvalidElementStateException\n        elif status in ErrorCode.INVALID_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:\n            exception_class = InvalidSelectorException\n        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:\n            exception_class = ElementNotSelectableException\n        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:\n            exception_class = ElementNotInteractableException\n        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:\n            exception_class = InvalidCookieDomainException\n        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:\n            exception_class = UnableToSetCookieException\n        elif status in ErrorCode.TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.SCRIPT_TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.UNKNOWN_ERROR:\n            exception_class = WebDriverException\n        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:\n            exception_class = UnexpectedAlertPresentException\n        elif status in ErrorCode.NO_ALERT_OPEN:\n            exception_class = NoAlertPresentException\n        elif status in ErrorCode.IME_NOT_AVAILABLE:\n            exception_class = ImeNotAvailableException\n        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:\n            exception_class = ImeActivationFailedException\n        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:\n            exception_class = MoveTargetOutOfBoundsException\n        elif status in ErrorCode.JAVASCRIPT_ERROR:\n            exception_class = JavascriptException\n        elif status in ErrorCode.SESSION_NOT_CREATED:\n            exception_class = SessionNotCreatedException\n        elif status in ErrorCode.INVALID_ARGUMENT:\n            exception_class = InvalidArgumentException\n        elif status in ErrorCode.NO_SUCH_COOKIE:\n            exception_class = NoSuchCookieException\n        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:\n            exception_class = ScreenshotException\n        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:\n            exception_class = ElementClickInterceptedException\n        elif status in ErrorCode.INSECURE_CERTIFICATE:\n            exception_class = InsecureCertificateException\n        elif status in ErrorCode.INVALID_COORDINATES:\n            exception_class = InvalidCoordinatesException\n        elif status in ErrorCode.INVALID_SESSION_ID:\n            exception_class = InvalidSessionIdException\n        elif status in ErrorCode.UNKNOWN_METHOD:\n            exception_class = UnknownMethodException\n        else:\n            exception_class = WebDriverException\n        if not value:\n            value = response['value']\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and 'message' in value:\n            message = value['message']\n    \n        screen = None  # type: ignore[assignment]\n        if 'screen' in value:\n            screen = value['screen']\n    \n        stacktrace = None\n        st_value = value.get('stackTrace') or value.get('stacktrace')\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split('\\n')\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = self._value_or_default(frame, 'lineNumber', '')\n                        file = self._value_or_default(frame, 'fileName', '<anonymous>')\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = self._value_or_default(frame, 'methodName', '<anonymous>')\n                        if 'className' in frame:\n                            meth = \"{}.{}\".format(frame['className'], meth)\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if 'data' in value:\n                alert_text = value['data'].get('text')\n            elif 'alert' in value:\n                alert_text = value['alert'].get('text')\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\nE         (unknown error: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x561b0c0b6cd3 <unknown>\nE       #1 0x561b0bebe968 <unknown>\nE       #2 0x561b0bee325c <unknown>\nE       #3 0x561b0bede8fa <unknown>\nE       #4 0x561b0bf1994a <unknown>\nE       #5 0x561b0bf13aa3 <unknown>\nE       #6 0x561b0bee93fa <unknown>\nE       #7 0x561b0beea555 <unknown>\nE       #8 0x561b0c0fe2bd <unknown>\nE       #9 0x561b0c102418 <unknown>\nE       #10 0x561b0c0e836e <unknown>\nE       #11 0x561b0c103078 <unknown>\nE       #12 0x561b0c0dcbb0 <unknown>\nE       #13 0x561b0c11fd58 <unknown>\nE       #14 0x561b0c11fed8 <unknown>\nE       #15 0x561b0c139cfd <unknown>\nE       #16 0x7ffa901b0609 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:247: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"start_session","time":{"start":1659439730786,"stop":1659439731548,"duration":762},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561b0c0b6cd3 <unknown>\n#1 0x561b0bebe968 <unknown>\n#2 0x561b0bee325c <unknown>\n#3 0x561b0bede8fa <unknown>\n#4 0x561b0bf1994a <unknown>\n#5 0x561b0bf13aa3 <unknown>\n#6 0x561b0bee93fa <unknown>\n#7 0x561b0beea555 <unknown>\n#8 0x561b0c0fe2bd <unknown>\n#9 0x561b0c102418 <unknown>\n#10 0x561b0c0e836e <unknown>\n#11 0x561b0c103078 <unknown>\n#12 0x561b0c0dcbb0 <unknown>\n#13 0x561b0c11fd58 <unknown>\n#14 0x561b0c11fed8 <unknown>\n#15 0x561b0c139cfd <unknown>\n#16 0x7ffa901b0609 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/autotetsamazon/autotetsamazon/conftest.py\", line 77, in start_session\n    Session.start()\n  File \"/home/runner/work/autotetsamazon/autotetsamazon/tests_utils/session.py\", line 27, in start\n    cls._driver = cls.get_new_driver()\n  File \"/home/runner/work/autotetsamazon/autotetsamazon/tests_utils/session.py\", line 43, in get_new_driver\n    driver = webdriver.Chrome(ChromeDriverManager().install(), options=options)\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 69, in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 92, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 277, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 370, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 435, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('start_session')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"tests_steps"},{"name":"subSuite","value":"TestScenario3orders"},{"name":"host","value":"fv-az462-697"},{"name":"thread","value":"2227-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.tests_steps"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"4d5ab36737c5924b","reportUrl":"https://Spirlhark.github.io/autotetsamazon/1//#testresult/4d5ab36737c5924b","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x560f33f0bcd3 <unknown>\n#1 0x560f33d13968 <unknown>\n#2 0x560f33d3825c <unknown>\n#3 0x560f33d338fa <unknown>\n#4 0x560f33d6e94a <unknown>\n#5 0x560f33d68aa3 <unknown>\n#6 0x560f33d3e3fa <unknown>\n#7 0x560f33d3f555 <unknown>\n#8 0x560f33f532bd <unknown>\n#9 0x560f33f57418 <unknown>\n#10 0x560f33f3d36e <unknown>\n#11 0x560f33f58078 <unknown>\n#12 0x560f33f31bb0 <unknown>\n#13 0x560f33f74d58 <unknown>\n#14 0x560f33f74ed8 <unknown>\n#15 0x560f33f8ecfd <unknown>\n#16 0x7fc7d4fe1609 <unknown>","time":{"start":1659438874791,"stop":1659438874791,"duration":0}}]},"tags":["@pytest.mark.usefixtures('start_session')"]},"source":"f841dbf164847c70.json","parameterValues":[]}